src/AcceptanceTests/App_Packages/NSB.AcceptanceTests.6.2.0/DelayedDelivery/When_Deferring_a_message.cs

```diff
@@ -29,7 +29,7 @@ public class When_deferring_a_message : NServiceBusAcceptanceTest
                 .Done(c => c.WasCalled)
                 .Run();
 
-            Assert.GreaterOrEqual(context.ReceivedAt - context.SentAt, delay);
+            Assert.That(context.ReceivedAt - context.SentAt, Is.EqualTo(delay).Within(1).Seconds);
```

Comment:
Wouldn't it be good enough to have two tests where we do the following:

- One test sets the delay to a fraction of the test run times and asserts that the message was received within the test runtime
- On test sets the delay to a multiple of the test run time and verifies that the message was not received within the test runtime

But not asserting the actual receivedAt or only that the timespan falls withing the test runtime which is actually automatically given when we received the message?